src/main/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoal.java

```diff
@@ -102,7 +102,11 @@ public static double calculateProgressTowardsReadingGoal(int toRead, int read) {
      *                       schedule
      * @return a String denoting that the user is ahead or behind schedule
      */
-    public static String behindOrAheadSchedule(int booksReadThisYear, int shouldHaveRead) {
-        return (booksReadThisYear < shouldHaveRead) ? BEHIND : AHEAD_OF;
+    public static String behindOrAheadSchedule(int booksReadThisYear, double shouldHaveRead) {
+        int decider = Double.compare(booksReadThisYear, shouldHaveRead);
+        if(decider == 1)
+            return AHEAD_OF;
+        else
+            return BEHIND;
```

Comment:
Could you replace this with a ternary operator?

Reply:
Done.